src/python_pybind11/test/Vector3_TEST.py

```diff
@@ -99,11 +101,11 @@ def test_vector3(self):
 
         self.assertEqual(Vector3d.ONE, Vector3d(1, 1, 1))
 
-        self.assertEqual(Vector3d.UNIT_X, Vector3d(1, 0, 0))
+        self.assertEqual(Vector3d.UNITX, Vector3d(1, 0, 0))
 
-        self.assertEqual(Vector3d.UNIT_Y, Vector3d(0, 1, 0))
+        self.assertEqual(Vector3d.UNITY, Vector3d(0, 1, 0))
 
-        self.assertEqual(Vector3d.UNIT_Z, Vector3d(0, 0, 1))
+        self.assertEqual(Vector3d.UNITZ, Vector3d(0, 0, 1))
```

Comment:
this seems like an API change. Is there a pythonic reason to not use `_` in a constant name?